apps/web/ee/components/LicenseRequired.tsx

```diff
@@ -0,0 +1,13 @@
+import { useSession } from "next-auth/react";
+import { FC } from "react";
+
+/**
+ * This component will only render it's children if the installation has a valid
+ * license.
+ */
+const LicenseRequired: FC = ({ children }) => {
+  const session = useSession();
+  return <>{session.data?.hasValidLicense ? children : null}</>;
```

Comment:
i would still render the children, if a license is missing, but disable it and give it opacity: 0.5, a cursor: not-allowed and a dialog of where to go to get a license

Reply:
Yeah it's still WIP